views/authentication/forms/login.hbs

```diff
@@ -98,14 +97,6 @@
                             </div>
                         </div>
                     </form>
-                    -->
```

Comment:
That change looks weird. But i just reverted the commit which do block comments around these divs